data/src/main/java/com/mument_android/data/repository/MumentDetailRepositoryImpl.kt

```diff
@@ -36,6 +41,12 @@ class MumentDetailRepositoryImpl @Inject constructor(
             .map {  }
             .toApiStatus(errorHandler)
 
+    override suspend fun reportMument(mumentId: String, reportRequest: ReportRequest): Void? {
+        reportMumentDataSource.reportMument(mumentId, reportMumentMapper.map(reportRequest)).let {
+            return it.data
```

Comment:
- 이렇게 boolean으로 내려줘도 좋을듯
```suggestion
            return it.status ==201
```

Reply:
옹 죠아영